### ISSUE TITLE
logistic function vs. logit 

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#69]
![](https://s3.amazonaws.com/cdn.graphpad.com/faq/1465/images/1465LogitLayout.png)

**Comment by WatershedXiaolan** [comment]
<img width="646" alt="Screen Shot 2022-02-25 at 4 13 07 PM" src="https://user-images.githubusercontent.com/5552371/155810192-143c29ff-5aa0-4e42-843f-e0381b072f7e.png">

**Comment by WatershedXiaolan** [comment]
The Logit function is used similarly to the sigmoid function in [neural networks](https://deepai.org/machine-learning-glossary-and-terms/neural-network). The sigmoid, or [activation, function](https://deepai.org/machine-learning-glossary-and-terms/activation-function) produces a probability, whereas the Logit function takes a probability and produces a real number between negative and positive infinity. Like the sigmoid function, Logit functions are often placed as the last layer in a neural network as can simplify the data. For example, a Logit function is often used in the final layer of a neural network used in classification tasks. As the network determines probabilities for classification, the Logit function can transform those probabilities to real numbers.
